packages/discovery-react-components/src/components/SearchFacets/utils/searchFilterTransform.tsx

```diff
@@ -93,10 +93,33 @@ export class SearchFilterTransform {
     const filterStrings: string[] = [];
     facets.forEach(facet => {
       const field = get(facet, 'field', '');
-      const results = get(facet, 'results', []);
-      const keys = this.quoteSelectedFacets(results, 'key');
-      if (keys.length) {
-        filterStrings.push(keys.map(key => `${escapeFieldName(field)}::${key}`).join('|'));
+      const results: SelectableQueryTermAggregationResult[] | never[] = get(facet, 'results', []);
+      const hasCategories = // logic duplicated from CollapsibleFacetsGroup
```

Comment:
If the logic is duplicated, it makes sense to refactor this into a utility method that can be used in both places. Makes code more maintainable and avoids issue where not all of the "instances" of this code are updated at the same time, in the future.